activerecord/lib/active_record/connection_adapters/mysql/schema_dumper.rb

```diff
@@ -53,7 +53,7 @@ def schema_limit(column)
           end
 
           def schema_precision(column)
-            super unless /\A(?:date)?time(?:stamp)?\b/.match?(column.sql_type) && column.precision == 0
+            super unless /\Atime(?:stamp)?\b/.match?(column.sql_type) && column.precision == 0
```

Comment:
Should this change to the following?

```suggestion
            super unless /\A(?:date)?time(?:stamp)?\b/.match?(column.sql_type) && column.precision == 6
```
Ie. do not dump the precision if it is the default. Ideally I'd have liked to refer to a constant (`eg. `DEFAULT_TIMESTAMP_PRECISION` somewhere, but unfortunately, the code is littered with the _[magic literal](https://en.wikipedia.org/wiki/Magic_number_(programming))_ `6`...

Reply:
In PostgreSQL, it always dumps the value of precision, and I thought it would be simpler to use the same specification for MySQL.
I think it would be better to always dump the precision value for `time` and `timestamp` types.

Reply:
Are there any reason not to dump the default value of precision?